### PR TITLE
docker port changes /4

### LINKED ISSUE #30971: docker image default port changed
### Description

Since .NET 8 docker image default port changed from 80 to 8080, should we mention it in the docker section

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/migration/70-80?view=aspnetcore-8.0&tabs=visual-studio#convert-a-blazor-server-app-into-a-blazor-web-app

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/migration/70-80.md

### Document ID

534ab561-09c5-c33b-4394-0effba34d666

### Article author

Rick-Anderson

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30971

[internal review url](https://review.learn.microsoft.com/en-us/aspnet/core/migration/70-80?view=aspnetcore-7.0&branch=pr-en-us-30989&tabs=visual-studio#docker)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/migration/70-80.md](https://github.com/dotnet/AspNetCore.Docs/blob/ff26797099b488115fe4e779091b572ed43f01b1/aspnetcore/migration/70-80.md) | [Migrate from ASP.NET Core 7.0 to 8.0](https://review.learn.microsoft.com/en-us/aspnet/core/migration/70-80?branch=pr-en-us-30989) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@WeihanLi please review

File: aspnetcore/migration/70-80.md [hunk 1]
```diff
@@ -438,15 +438,25 @@ Updated configuration guidance appears in the following locations:
...
 ```
 
+### Update Docker port
+
+The default ASP.NET Core port configured in .NET container images has been updated from port 80 to 8080.
+
+The new `ASPNETCORE_HTTP_PORTS` environment variable was added as a simpler alternative to `ASPNETCORE_URLS`.
+
+For more information, see [Default ASP.NET Core port changed from 80 to 8080](/dotnet/core/compatibility/containers/8.0/aspnet-port).
+
```
Comment (Rick-Anderson) [inline aspnetcore/migration/70-80.md:459]: This is the only change. GitHub diff can't deal with `-+`

File: aspnetcore/migration/70-80.md [hunk 2]
```diff
@@ -438,15 +438,25 @@ Updated configuration guidance appears in the following locations:
...
 ```diff
 - docker pull mcr.microsoft.com/dotnet/aspnet:7.0
 + docker pull mcr.microsoft.com/dotnet/aspnet:8.0
 ```
 
+### Update Docker port
+
+The default ASP.NET Core port configured in .NET container images has been updated from port 80 to 8080.
+
+The new `ASPNETCORE_HTTP_PORTS` environment variable was added as a simpler alternative to `ASPNETCORE_URLS`.
```
Comment (tdykstra) [inline aspnetcore/migration/70-80.md:456]: You could link to https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-8.0#specify-ports-only for more information.

**Review by tdykstra — APPROVED** [review]
Looks good, one link suggestion.

**Pull request merged**